observability-charter.md

```diff
@@ -0,0 +1,207 @@
+# CNCF SIG Observability Charter
+
+- [CNCF SIG Observability Charter](#cncf-sig-observability-charter)
+  - [Introduction](#introduction)
+  - [Mission](#mission)
+  - [Areas considered in Scope](#areas-considered-in-scope)
+  - [Areas considered out of Scope](#areas-considered-out-of-scope)
+  - [Roadmap & Initial Efforts](#roadmap--initial-efforts)
+  - [Governance](#governance)
+  - [Operations](#operations)
+
+*Initially authored by [Matthew Young][matthew young] with grateful review and
+contributions from:
+[Alex Nauda][Alex Nauda],
+[Alois Reitbauer][Alois Reitbauer],
+[Bartłomiej (Bartek) Płotka][Bartłomiej (Bartek) Płotka],
+[Daniel Khan][Daniel Khan],
+[Daniel Prata][Daniel Prata],
+[Lincoln Sward][Lincoln Sward],
+[Matthias Loibl][Matthias Loibl],
+[Michael Hausenblas][Michael Hausenblas],
+[Ricardo Aravena][Ricardo Aravena],
+[Richard Hartmann][Richard Hartmann],
+[Sergey Kanzhelev][Sergey Kanzhelev],
+[Steve Flanders][Steve Flanders],
+[Ted Young][Ted Young],
+[Tigran Najaryan][Tigran Najaryan],
+[Tommy Chong][Tommy Chong],
+and [Umair Ishaq][Umair Ishaq].*
+
+<!-- TODO: please put github names here -->
+[Matthew Young]:              https://github.com/halcyondude
+[Alex Nauda]:                 @
+[Alois Reitbauer]:            https://github.com/aloisreitbauer
+[Bartłomiej (Bartek) Płotka]: https://github.com/bwplotka
+[Daniel Khan]:                @
+[Daniel Prata]:               @
+[Lincoln Sward]:              @
+[Matthias Loibl]:             https://github.com/metalmatze
+[Michael Hausenblas]:         https://github.com/mhausenblas
+[Ricardo Aravena]:            https://github.com/raravena80
+[Richard Hartmann]:           https://github.com/RichiH
+[Sergey Kanzhelev]:           @
+[Steve Flanders]:             https://github.com/flands
+[Ted Young]:                  @
+[Tigran Najaryan]:            @
+[Tommy Chong]:                https://github.com/techietommy
+[Umair Ishaq]:                https://github.com/umairishaq
+
+## Introduction
+
+This document describes the purpose and operations of the Cloud Native
+Computing Foundation ([CNCF]) Special Interest Group ([SIG]) on Observability.
+
+This [SIG] focuses on topics pertaining to the observation
+of [cloud native][cn-def] workloads. Additionally, it produces supporting
+material and best practices for end-users and provides guidance and
+coordination for CNCF projects working within the SIG’s scope.
+
+A full list of [CNCF projects][projs] can be found at [landscape.cncf.io].
```

Comment:
Nice, it would be even nicer if the `https://landscape.cncf.io/ website would allow filtering through Categories where each SIG is a different category! :hugs:

Reply:
Perfect example of the kind of thing we can drive once the SIG is up and running :)